# what type of information ()
# 1: Message
# 2: Motion
int32 kind
# for Motion: motion name + args
# for Message: Message label
string description
# Motion / Message parameters
float32[] parameters
